server/services/bigtable.py

```diff
@@ -87,7 +88,7 @@ async def write_row(session_info, data):
   row.set_cell(_COLUMN_FAMILY, _COL_PROJECT.encode(), project_id)
   row.set_cell(_COLUMN_FAMILY, _COL_SESSION.encode(), json.dumps(session_info))
   row.set_cell(_COLUMN_FAMILY, _COL_VERSION.encode(), json.dumps(version))
-  row.set_cell(_COLUMN_FAMILY, _COL_DATA.encode(), json.dumps(data))
+  row.set_cell(_COLUMN_FAMILY, _COL_DATA.encode(), pickle.dumps(data))
```

Comment:
Maybe revert just this change, and I'll make a PR as followup?

Reply:
I will hold off this PR for other purpose then, since this was aimed to fix this bug.

Reply:
No, I don't think its from the recent change that added MultiSV.  Generally data_dict is not safe to convert to json.

So submit this change unbreak autopush for now, and I can look into why this is and switch to json?

Reply:
sg!